apps/Cargo.toml

```diff
@@ -74,6 +74,7 @@ abciplus = [
 
 [dependencies]
 namada = {path = "../shared", default-features = false, features = ["wasm-runtime", "ferveo-tpke"]}
+namada_ethereum_bridge = {path = "../ethereum_bridge"}
```

Comment:
The pattern here is to re-export needed deps inside of the shared sub-crate. You'll notice that PoS does the same.

Reply:
oic, have updated to use `namada::eth_bridge` instead

Reply:
updated in `tests` crate also now